exercises/bracket-push/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// The code below is a stub. Just enough to satisfy the compiler.
+// In order to pass the tests you can add-to or change any of this code.
+
+pub struct Brackets;
+
+impl<'a> From<&'a str> for Brackets {
+    fn from(i: &str) -> Self {
```

Comment:
is `s` better? or does the `i` stand for "input"?

I don't actually care, but I ask in case it matters to someone